pkg/reconciler/workload/namespace/namespace_reconcile.go

```diff
@@ -262,18 +269,29 @@ func (c *Controller) reconcileNamespace(ctx context.Context, lclusterName logica
 // enqueueResourcesForNamespace adds the resources contained by the given
 // namespace to the queue if there scheduling label differs from the namespace's.
 func (c *Controller) enqueueResourcesForNamespace(ns *corev1.Namespace) error {
+	clusterName := logicalcluster.From(ns)
+
 	nsLocation := ns.Labels[ClusterLabel]
 
+	klog.V(4).Infof("enqueueResourcesForNamespace(%s|%s): getting listers", clusterName, ns.Name)
```

Comment:
We can start proliferating structured logs at any point, this might be

```go
klog.V(4).InfoS("getting listers", "method", "enqueueResourcesForNamespace", "clusterName", clusterName, "namespace", ns.Name)
```